ui/helpers/utils/i18n-helper.tsx

```diff
@@ -96,7 +101,7 @@ export const getMessage = (localeCode, localeMessages, key, substitutions) => {
   return phrase;
 };
 
-export async function fetchLocale(localeCode) {
+export async function fetchLocale(localeCode: string): Promise<any> {
```

Comment:
I believe the return type of this should be `I18NMessageDict` from above.

Reply:
Gotcha !